profile-page.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Profilepage</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="profile-page.css" rel="stylesheet" type="text/css"> 
+       </div>
+    
+    </head>
+
+<body>
+    <div class="picture">
+
+        <h1>Catherine Butler</h1>
+    </div>
+    <meta>
+
+   <div class="row">
+       <div class="side">
+        <div style="text-align: left"></div><img src="Headshot.jpg" alt="cat" style="width:100%">
+        <br>
+
+        <p style="font-size: 30px; text-align: center; width:100%; color: purple;">
```

Comment:
It can be better to separate the concerns - split html and css so your styles are in your style sheet, this is easier to manage and maintain